region/lowernorfair/east.json

```diff
@@ -1576,415 +1975,775 @@
+                  ],
+                  "obstacles": [
+                    {
+                      "id": "C",
+                      "requires": [
+                        {"enemyKill":{
+                          "enemies": [
+                            [ "Dessgeega" ]
+                          ],
+                          "excludedWeapons": [ "Bombs" ]
+                        }}
+                      ]
+                    }
+                  ],
+                  "note": "Morph Bombs excluded because that'd probably cost more health than just running through."
+                },
+                {
+                  "name": "Screw Kill",
+                  "notable": false,
+                  "requires": [
+                    "canNavigateHeatRooms",
+                    {"heatFrames": 80}
+                  ],
+                  "obstacles": [
+                    {
+                      "id": "C",
+                      "requires": [
+                        {"heatFrames": 50},
+                        "ScrewAttack"
+                      ]
+                    }
+                  ]
+                },
+                {
+                  "name": "Supers Kill",
+                  "notable": false,
+                  "requires": [
+                    "canNavigateHeatRooms",
+                    {"heatFrames": 80}
+                  ],
+                  "obstacles": [
+                    {
+                      "id": "C",
+                      "requires": [
+                        {"heatFrames": 50},
+                        {"enemyKill":{
+                          "enemies": [
+                            [
+                              "Dessgeega"
+                            ]
+                          ],
+                          "explicitWeapons": [ "Super" ]
+                        }}
+                      ]
+                    }
+                  ]
+                },
+                {
+                  "name": "Beam Kill",
+                  "notable": false,
+                  "requires": [
+                    "canNavigateHeatRooms",
+                    {"heatFrames": 80}
+                  ],
+                  "obstacles": [
+                    {
+                      "id": "C",
+                      "requires": [
+                        {"heatFrames": 50},
+                        "Plasma",
+                        {"or": [
+                          "Ice",
+                          "Wave"
+                        ]}
+                      ]
+                    }
+                  ]
+                }
+              ],
+              "note": "No strat specifically for if the enemy is just dead, since all the different kill strats require nothing if it's already dead."
+            }
+          ]
+        },
+        {
+          "from": 7,
+          "to": [
+            {
+              "id": 2,
+              "strats": [
+                {
+                  "name": "Avoid Hibashi",
```

Comment:
Roll past the spike and springball bounce up might be reasonable.
Kaizo bombjump works
springball out of the tunnel
roll out, unmorph and jump is probably too precise.
mochball roll through and pray

Reply:
Are you suggesting those as other alternatives that also require heatproof? I feel like expecting those in a suitless scenario adds a lot of difficulty for the 30 energy it saves, which you can lose in just 2 seconds of hesitation.